pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -393,9 +399,9 @@ func (c *VPCConfig) getRulesOfConnection(src, dst Node,
 	return allowRulesOfConnection, denyRulesOfConnection, nil
 }
 
-func (rulesInLayers rulesInLayers) updateRulesPerLayerIfNonEmpty(layer string, rulesFilter *[]RulesInTable) {
+func (rules rulesInLayers) updateRulesPerLayerIfNonEmpty(layer string, rulesFilter *[]RulesInTable) {
```

Comment:
"who is the receiver" comment

Reply:
This function main context is the receiver